frontend/src/components/dashboard/aliases/AddressPickerModal.tsx

```diff
@@ -189,7 +188,6 @@ type PickerDialogProps = {
 const PickerDialog = (props: PickerDialogProps & AriaOverlayProps) => {
   const wrapperRef = useRef<HTMLDivElement>(null);
   const { overlayProps, underlayProps } = useOverlay(props, wrapperRef);
-  usePreventScroll();
```

Comment:
@Vinnl So I discovered that the reason it was scrolling up in a strange fashion on Safari was due to this `usePreventScroll()` hook. I removed it and I didn't see a noticeable difference in UI when checked across the desktop and the simulators. Could you explain in detail what this is doing for the component or is it imperative to leave it in?

Reply:
Hmm, interesting. Looking at [its docs](https://react-spectrum.adobe.com/react-aria/usePreventScroll.html), it seems to have been created specifically for `useDialog`, [whose documentation](https://react-spectrum.adobe.com/react-aria/useDialog.html#example) no longer mentions it. It's not even mentioned in the sidebar of that page, so I've [logged a bug upstream](https://github.com/adobe/react-spectrum/issues/3909), but I'm guessing that indeed it is no longer necessary?

The scrolling behaviour in your preview deployment when opening the custom mask dialog seems normal to me, so let's indeed remove it and keep an eye on QA reports related to scrolling/the custom mask dialog.